package.json

```diff
@@ -27,12 +27,26 @@
     "url": "https://github.com/lob/hapi-bookshelf-total-count/issues"
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
+  "peerDependencies": {
+    "bookshelf": "0.9.x",
+    "hapi": "11.x"
+  },
```

Comment:
These peer dependencies are more restrictive than I would want but we can support more versions when a need arises.